api/checks/webhook.go

```diff
@@ -193,21 +223,26 @@ func handleCheckRunEvent(
 		return false, nil
 	}
 
+	// Experimental support for consuming Azure Pipelines results via the
+	// GitHub Checks API.
+	if appID == azure.PipelinesAppID {
+		if aeAPI.IsFeatureEnabled("processAzureCheckRunEvents") {
+			return azureAPI.HandleCheckRunEvent(checkRun)
+		}
+		log.Infof("Ignoring Azure pipelines event")
+		return false, nil
+	}
+
 	login := checkRun.GetSender().GetLogin()
 	if !isUserWhitelisted(aeAPI, login) {
 		log.Infof("Sender %s not whitelisted for wpt.fyi checks", login)
 		return false, nil
 	}
 
+	// TODO: Explain all this, especially what a requested_action is.
```

Comment:
I think this is about https://developer.github.com/v3/checks/runs/#check-runs-and-requested-actions, the "Ignore" and "Recompute" actions we have on each check.

The "Ignore" bit would be useful if these checks could block, but we backed down from that in order to improve the heuristics a lot before going there again.